lib/src/presentation/views/home.dart

```diff
@@ -1,20 +1,321 @@
+                                      Row(
+                                        mainAxisAlignment:
+                                            MainAxisAlignment.spaceAround,
+                                        children: <Widget>[
+                                          Text(
+                                            StringConstants.dairyFreeTag,
+                                            style: tagsStyle(
+                                              recipes[index].dairyFree,
+                                            ),
+                                          ),
+                                          Text(
+                                            StringConstants.veryHealthyTag,
+                                            style: tagsStyle(
+                                              recipes[index].veryHealthy,
+                                            ),
+                                          ),
+                                          Text(
+                                            StringConstants.cheapTag,
+                                            style: tagsStyle(
+                                              recipes[index].cheap,
+                                            ),
+                                          ),
+                                        ],
+                                      ),
+                                      Row(
+                                        mainAxisAlignment:
+                                            MainAxisAlignment.spaceAround,
+                                        children: <Widget>[
+                                          Text(
+                                            StringConstants.veryPopularTag,
+                                            style: tagsStyle(
+                                              recipes[index].veryPopular,
+                                            ),
+                                          ),
+                                          Text(
+                                            StringConstants.sustainableTag,
+                                            style: tagsStyle(
+                                              recipes[index].sustainable,
+                                            ),
+                                          ),
+                                          Text(
+                                            StringConstants.lowFodmapTag,
+                                            style: tagsStyle(
+                                              recipes[index].lowFodmap,
+                                            ),
+                                          )
+                                        ],
+                                      ),
+                                    ],
+                                  ),
+                                ],
+                              ),
+                            ),
+                          );
```

Comment:
Maybe you can create a new widget you are replicating the row three times

Reply:
Done